src/utilities/scroll-lock-manager/hooks.tsx

```diff
@@ -4,5 +4,10 @@ import {ScrollLockManagerContext} from './context';
 export function useScrollLockManager() {
   const scrollLockManager = useContext(ScrollLockManagerContext);
 
+  if (!scrollLockManager) {
+    throw new Error(
+      'No ScrollLockManager was provided. Your application must be wrapped in an <AppProvider> component. See https://polaris.shopify.com/components/structure/app-provider for implementation instructions.',
```

Comment:
Should error messages follow the same standards as the [console messages](https://github.com/Shopify/polaris-react/blob/master/documentation/Console%20messages.md)?

Reply:
They contain the message, and then a link to find out more. What more do you think is missing. Prefixing the string with "Error" seems somewhat pointless.

Reply:
I initially thought we prefixed messages with `[Polaris]` (for easy filtering) but it seems like the guidelines don't mention anything about that.